app/components/common/navbar/index.tsx

```diff
@@ -139,8 +144,9 @@ const Navbar = () => {
         <ul
           className={`hidden md:flex flex-col ${classShortHands.padding} mt-4 md:mt-0 md:text-sm md:font-medium md:border-0 ${classShortHands.navbarBackgroundColor}`}
         >
-          {navbarElements.navbarSettings.map((navIntem: NavItemType) => (
+          {navbarElements.navbarSettings.map((navIntem: NavItemType, index: number) => (
             <li
+              key={index}
```

Comment:
we can use the `id` available in `navItem` instead of index

Reply:
Done.